.azure-pipelines/code-oob.yml

```diff
@@ -99,6 +73,50 @@ stages:
           $readmePath = "OOB/README.md"
           $content | Out-File -FilePath $readmePath
 
+          if ($NeedBumpUp) {
+              echo "##vso[task.setvariable variable=BumpUpBranch;isOutput=true]codeoob/${{ parameters.ServiceName }}"
+          } else {
+              echo "##vso[task.setvariable variable=BumpUpBranch;isOutput=true]none"
+          }
+    - task: DotNetCoreCLI@2
+      displayName: Build
+      condition: and(succeeded(), eq(variables['NeedBumpUp'], 'true'))
+      inputs:
+        command: custom
+        custom: msbuild
+        arguments: 'build.proj "/t:Clean;Build" /p:Configuration=Release'
+    - pwsh: |
+          .\tools\RunVersionController.ps1 -ModuleName "Az.${{ parameters.ServiceName }}”
+      displayName: 'Bump up version'
+      condition: and(succeeded(), eq(variables['NeedBumpUp'], 'true'))
+    - task: AzurePowerShell@5
+      inputs:
+          azureSubscription: '$(AzureSubscription)'
+          ScriptType: 'InlineScript'
+          azurePowerShellVersion: 'LatestVersion'
+          inline: |
+              Set-AzContext -Tenant $(TenantId) -SubscriptionId $(SubscriptionId)
+              $Token = Get-AzKeyVaultSecret -VaultName $(KeyVaultName) -Name $(KeyVaultAccount) -AsPlainText
+              git config user.email "65331932+azure-powershell-bot@users.noreply.github.com"
+              git config user.name "azure-powershell-bot"
+              git checkout -b codeoob/${{ parameters.ServiceName }}

```

Comment:
Make sure you reuse `BumpUpBranch` instead of hardcode in this script

Reply:
BumpUpBranch is an output variable for another stage or job. As I use it in the next stage, it should be $[stageDependencies.OOBRelease.BumpUpVersion.outputs['outputinfo.BumpUpBranch']]. Actually more complicated